packages/ts-codegen/src/__tests__/index.test.ts

```diff
@@ -527,14 +515,15 @@ describe(generateTS, () => {
             readonly __typename?: "User";
             readonly id: string;
         }
+        export type UserParamsInput = {
+            readonly name?: string | null;
+        };
         "
       `);
       expect(resolvers).toMatchInlineSnapshot(`
         "import type { PromiseOrValue } from "@graphitation/supermassive";
         import type { ResolveInfo } from "@graphitation/supermassive";
         import * as Models from "./models.interface";
-        import * as Inputs from "./inputs.interface";
-        export * from "./inputs.interface";
```

Comment:
I wonder if these exports were used. Should I `export * from "./models.interface"` to cover for it? 🤔

Reply:
skimmed through TMP codebase and there doesn't seem to be usages for this particular export. There's only a few instance of direct imports from `inputs.interface` but those look easy enough to fix.

If there's no issues with the change itself, I can prepare a patch to test if TMP CI breaks at the very least.

Reply:
Tested latest changes with TMP. Added tests for missing cases.